addon/runtime.js

```diff
@@ -34,6 +34,9 @@ export default function(addon) {
       // apply defaults
       optparse.mergeTrackerOpts(mapping, _settings);
       optparse.handlerOpts(mapping);
+      optparse.timingHandlerOpts(mapping);
```

Comment:
I guess the `timingHandlerOpts` and others should be available both for `configure` and `track` and pushed into global `chains` which is available for particular environment.